cartographer/mapping_2d/submaps.cc

```diff
@@ -151,6 +151,21 @@ void Submap::ToResponseProto(
       transform::Rigid3d::Translation(Eigen::Vector3d(max_x, max_y, 0.)));
 }
 
+void Submap::InsertRangeData(const sensor::RangeData& range_data,
+                             const RangeDataInserter& range_data_inserter) {
+  CHECK(!finished_);
+  range_data_inserter.Insert(range_data, &probability_grid_);
+  ++num_range_data_;
+}
+
+void Submap::Finish() {
+  CHECK(!finished_);
+  // Crop the finished Submap before inserting a new Submap to reduce peak
```

Comment:
I think this comment is not understandable here. Move next to the call to `FinishSubmap()` in `AddSubmap()`?